### PR TITLE
[Main Course] - Graph Database Introduction

### PR DESCRIPTION
**Pull request by dragond041**
The first pull request of Introduction of Graph Database. Feel free to give me your comments, thanks!

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# How Has This Been Tested?

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by dragond041 — COMMENTED** [review]
I'll adjust the figure size and add caption later.

File: docs/Main_Course/Graph_Database/Graph_Database_Introduction.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+template: overrides/blogs.html
+---
+
+# 初识图数据库
+
+!!! info
+    作者：Dragond41，发布于2021-12-13，阅读时间：约2分钟，微信公众号文章链接：[:fontawesome-solid-link:]()
+
+## 1 前言
```
Comment (EchoTang04) [inline docs/Main_Course/Graph_Database/Graph_Database_Introduction.md:10]: 1后面加“.” 跟后面的每一大点格式保持一致

File: docs/Main_Course/Graph_Database/Graph_Database_Introduction.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+<p>上图中的圆形图案都是节点。其中橙色的节点代表是电影实体，蓝色的节点代表的是具体的人。
+
+### 2.2 关系
+两个节点之间的联系就是关系。如上述图中就出现了三种【人-电影】的关系：
+- <font size="1">ACTED_IN - 基努·里维斯、劳伦斯·菲什伯恩、凯莉·安摩丝和雨果·维文等参演
+- <font size="1">DIRECTED - 沃卓斯基姐妹指导
+- <font size="1">PRODUCED - 乔·西佛监制
+
+### 2.3 关系的方向
+在Neo4J中，关系是必须有方向的。
```
Comment (EchoTang04) [inline docs/Main_Course/Graph_Database/Graph_Database_Introduction.md:38]: 可以加一句Neo4J的解释。比如是一个世界领先的开源图形数据库等等。

**Review by EchoTang04 — APPROVED** [review]
LGTM, thanks for sharing~

File: docs/Main_Course/Graph_Database/Graph_Database_Introduction.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+  <figure>
+  <img src="https://cdn.jsdelivr.net/gh/BulletTech2021/Pics/img/1_V3.png"  />
+  <figcaption></figcaption>
+</figure>
+
+## 3. Why图数据库
+大家看了对图数据库的大体介绍，可能觉得这个技术很酷炫，那在具体的业务中，图数据库能带来哪些关系型数据库不能带来的价值呢？以下两方面可以作为参考。
+
+- <font size="1">高效的查询数据间的关系，尤其是当关系很复杂时
```
Comment (EchoTang04) [inline docs/Main_Course/Graph_Database/Graph_Database_Introduction.md:62]: 感觉可以更加详细的介绍一下相比于关系型数据库的优点。FYI：https://zhuanlan.zhihu.com/p/50171330 这里面查找某个公司的员工属于哪个部门的例子也蛮好。

File: docs/Main_Course/Graph_Database/Graph_Database_Introduction.md (hunk 1 above)
Comment (Realvincentyuan) [reply docs/Main_Course/Graph_Database/Graph_Database_Introduction.md:10]: 我们之前的一级标题后面没有点的，可以直接用1 标题，把后面的改了吧

File: docs/Main_Course/Graph_Database/Graph_Database_Introduction.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+</figure>
+  
+当然没有人会从这样的可视化结果中获取有价值的信息，那图数据库具体应该如何应用在业务中呢？
+<p>请期待之后的图数据库实战系列，绝对能让你对这个又新又潮的技术有更具体的认识！
+  
+    
+## 4. 总结
+  
+学习了本篇文章之后是不是不会被忽悠什么是图数据库了（狗头），而且能在朋友面前吹吹牛呢？
+<p>如果想继续收看和图数据相关的文章的话，就点个赞或者在看吧！
```
Comment (Realvincentyuan) [inline docs/Main_Course/Graph_Database/Graph_Database_Introduction.md:88]: 这些HTML的标签请去除哦

**Review by Realvincentyuan — COMMENTED** [review]
谢谢贡献！有些评论

**Review by jhqian0909 — APPROVED** [review]

File: docs/Main_Course/Graph_Database/Graph_Database_Introduction.md (hunk 1 above)
Comment (dragond041) [reply docs/Main_Course/Graph_Database/Graph_Database_Introduction.md:10]: 已经把 点 去掉了

File: docs/Main_Course/Graph_Database/Graph_Database_Introduction.md (hunk 4 above)
Comment (dragond041) [reply docs/Main_Course/Graph_Database/Graph_Database_Introduction.md]: 已经把<p>替换成空行了

**Review by Tina0330 — APPROVED** [review]
感谢分享！

**Pull request merged**